acceptance-tests/src/test/resources/features/UKDrivingLicence/WelshLangDrivingLicence.feature

```diff
@@ -13,6 +13,15 @@ Feature: Driving License Language Test
     When the beta banner reads Mae hwn yn wasanaeth newydd – bydd eich adborth (agor mewn tab newydd) yn ein helpu i'w wella.
     Then The test is complete and I close the driver
 
+  @Language-regression
+  Scenario: Beta Banner Reject Analysis button
+    Given I view the Beta banner
+    When I select Gwrthod cwcis dadansoddi cookie
+    Then I see the Reject Analysis sentence Rydych wedi gwrthod cwcis ychwanegol. Gallwchnewid eich gosodiadau cwcis" unrhyw bryd.
```

Comment:
Looks like speech marks in the wrong place here I see it in the code as well but can we confirm with the BAs if this right